[1.9+] Fixed incorrect information on resiliency policies (#3153)
dapr/docs: daprdocs/content/en/operations/resiliency/targets.md

With the `actors` target, you can apply `retry`, `timeout`, and `circuitBreaker` policies to actor operations. 

> When using a `circuitBreaker` policy, you can specify whether circuit breaking state should be scoped to by using `circuitBreakerScope`:

Reviewer: ```suggestion
When using a `circuitBreaker` policy for the `actors` target, you can specify how circuit breaking state should be scoped by using `circuitBreakerScope`:
```
